directory/src/main/java/org/ecclesia/directory/domain/Organization.java

```diff
@@ -3,14 +3,16 @@
 import javax.persistence.*;
 import javax.validation.constraints.NotBlank;
 import java.io.Serializable;
```

Comment:
**Reporter**: Checkstyle
**Rule**: com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck
**Severity**: WARN
**File**: /var/jenkins_home/workspace/ecclesia_PR-7/directory/src/main/java/org/ecclesia/directory/domain/Organization.java L5

Wrong lexicographical order for &#39;java.io.Serializable&#39; import. Should be before &#39;javax.validation.constraints.NotBlank&#39;.


*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-594961164>*